assets/l10n/intl_en.arb

```diff
@@ -2927,5 +2927,30 @@
   "contactsCount": "Contacts ({count})",
   "linagoraContactsCount": "Linagora contacts ({count})",
   "fetchingPhonebookContacts": "Fetching contacts from device...({progress}% completed)",
-  "activeNow": "Active now"
+  "activeNow": "Active now",
+  "languageEnglish": "English",
+  "@languageEnglish": {
+    "type": "text",
+    "placeholders_order": [],
+    "placeholders": {}
+  },
+  "languageVietnamese": "Vietnamese",
```

Comment:
you should use this lib to get local name from local codes
https://pub.dev/packages/flutter_localized_locales
we also need support all languages defined in `L10n.supportedLocales`

Reply:
Don't need to use lib. Something so simple depends on an overly complicated lib

Reply:
I prefer to use lib in this case. It is simple now, but it can more complicate in the future.